cypress/integration/account_settings/theme_color/custom_colors_sidebar_styles_spec.js

```diff
@@ -117,9 +117,6 @@ describe('AS14318 Theme Colors - Color Picker', () => {
 
     describe('Custom - Sidebar styles target output change', () => {
         it('should take effect each custom color in Channel View', () => {
-            // * Check Sidebar Header Text color change after saving
-            cy.get('#accountSettingsModalLabel').should('have.css', 'color', 'rgb(129, 65, 65)');
```

Comment:
The preceding test already tested this.  It's failing since the preceding also closes the account settings modal.